dexs/tonpump/index.ts

```diff
@@ -0,0 +1,32 @@
+import fetchURL from "../../utils/fetchURL"
+import { CHAIN } from "../../helpers/chains";
+import { FetchOptions } from "../../adapters/types";
+import { getPrices } from "../../utils/prices";
+
+
+const endpoint = "https://tonfunstats-eqnd7.ondigitalocean.app/api/v1/getVolume"
+
+
+const fetch = async (options: FetchOptions) => {
+  const res = await fetchURL(`${endpoint}?from=${options.startTimestamp}&to=${options.endTimestamp}&service=hot`)
+  const TON = "coingecko:the-open-network"
+  const ton_price = await getPrices([TON], options.startTimestamp);
```

Comment:
you dont need to fetch ton price, can return as `{'the-open-network': res.volume/1e9}`